tests/samples/matmul_fill_static.mlir

```diff
@@ -1,9 +1,23 @@
-func.func @matmul_static(%lhs : tensor<128x256xf32>,
-    %rhs : tensor<256x512xf32>) -> tensor<128x512xf32> {
-  %empty = tensor.empty() : tensor<128x512xf32>
-  %cst = arith.constant 0.0 : f32
-  %fill = linalg.fill ins(%cst : f32) outs(%empty : tensor<128x512xf32>) -> tensor<128x512xf32>
-  %2 = linalg.matmul ins(%lhs, %rhs : tensor<128x256xf32>, tensor<256x512xf32>)
-      outs(%fill : tensor<128x512xf32>) -> tensor<128x512xf32>
-  return %2 : tensor<128x512xf32>
+// Example using a transform dialect script.
+
+// RUN: iree-opt %s   --iree-hal-target-backends=amd-aie \
+// RUN:               --iree-abi-transformation-pipeline \
+// RUN:              --iree-flow-transformation-pipeline \
+// RUN:            --iree-stream-transformation-pipeline \
+// RUN:                --iree-hal-configuration-pipeline \
+// RUN: | cat -  $(dirname %s)/matmul_fill_spec_pad.mlir \
+// RUN: | iree-opt  --iree-transform-dialect-interpreter \
```

Comment:
Interesting, I think this approach expects the transform dialect script to be in the same file. I havent seen this usage mode.

Reply:
Yes, that's why I used `cat`.